CAFAna/PRISM/app/OffAxisNDCAFCombiner.C

```diff
@@ -121,6 +121,15 @@ std::vector<std::string> GetMatchingFiles(std::string directory,
   return matches;
 }
 
+void pnfs_to_xrootd(std::string &dir) {
```

Comment:
A function very like this one already exists in cafanacore
https://github.com/cafana/CAFAnaCore/blob/master/CAFAna/Core/UtilsExt.cxx#L237
I don't remember all the ins and outs of the `unauth` business, but it was important at some point, and the default behaviour is the same as you have here.

Reply:
`#include "CAFAna/Core/UtilsExt.h"`

Reply:
In the PRISM branch of lblpwgtools, the function you mention still lives in CAFAna/Core/Utilities.cxx and is slightly different. The path returned by the PRISM branch version for authed onsite is here 

https://github.com/DUNE/lblpwgtools/blob/80fb38ce0d111aa80874a7ad1770b94c84a5a718/CAFAna/Core/Utilities.cxx#L871

When trying to open a file using this path, root just hangs. Going to change the line mentioned above to return the path given in CAFAna/Core/UtilsExt.cxx, will then remove pnfs_to_xrootd from OffAxisNDCAFCombiner.C, instead including CAFAna/Core/Utilities.h and using pnfs2xrootd as you suggested.